src/components/Jobs/components/Job/Job.css

```diff
@@ -20,6 +20,15 @@
   }
 }
 
+[class*="listStyleDefault"] {
```

Comment:
Will it work for `.listStyleDefault`?

Reply:
This styles apply only for `.job` inside `. listStyleDefault ` and it's only this case.

Reply:
I mean will it work if `[class*="listStyleDefault"]` replace with `.listStyleDefault`?

Reply:
no, because of hash added to styles for parent component

Reply:
Got it, thanks